pkg/apis/migration.harvesterhci.io/v1beta1/virtualmachines.go

```diff
@@ -93,3 +100,7 @@ const (
 	VirtualMachineExportFailed    condition.Cond = "VMExportFailed"
 	VirtualMachineMigrationFailed ImportStatus   = "VMMigrationFailed"
 )
+
+func (in *VirtualMachineImport) GetDefaultDiskBusType() kubevirtv1.DiskBus {
+	return ptr.Deref[kubevirtv1.DiskBus](in.Spec.DefaultDiskBusType, kubevirtv1.DiskBusVirtio)
```

Comment:
we do not need the explicity type argumet since `vm.Spec.DefaultDiskBusType` is already `kubevirtv1.DiskBus`

Reply:
No, but I just want to make use of the “default” functionality of the defer function that I get here for free.